src/DssSpell.sol

```diff
@@ -19,21 +19,24 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
+import { GemAbstract } from "dss-interfaces/ERC/GemAbstract.sol";
```

Comment:
Nit: this is usually imported like:
```suggestion
import from "dss-interfaces/ERC/GemAbstract.sol";
```

Since the files only export 1 single interface, it is fine to use a global import.

Reply:
But that's exactly what checklist [is recommending](https://github.com/makerdao/pe-checklists/blob/81ed24f37b477a07bb59abd8cd60f6049ad4fc1c/spell/spell-crafter-mainnet-workflow.md?plain=1#L90)

Reply:
I don't see a reason why a more specific import using Curly Brackets would be less preferred over a full file import, even if they have the same effect in the end. More explicit is always better in my opinion

Reply:
The [reviewer's checklist](https://github.com/makerdao/pe-checklists/blob/81ed24f37b477a07bb59abd8cd60f6049ad4fc1c/spell/spell-reviewer-mainnet-checklist.md?plain=1#L43) shows the global import.

I don't mind being explicit, just flagging here because of the checklist.

If we want to use this format moving forward, let's update the reviewer's checklist so they are not in conflict.

Reply:
I think this part looks a bit confusing in reviewer checklist. Let's align this separately :) 

https://github.com/makerdao/pe-checklists/blob/81ed24f37b477a07bb59abd8cd60f6049ad4fc1c/spell/spell-reviewer-mainnet-checklist.md?plain=1#L43